CONTRIBUTING.md

```diff
@@ -43,22 +43,23 @@ We will only accept a pull request for which all tests pass. Make sure the follo
 
 ## Getting started
 
```

Comment:
It's probably a good idea to link to a general guide about forking. https://help.github.com/en/articles/fork-a-repo looks quite nice and is not on a 3rd party website. It's tempting to link from this from the first step but I'd like to make this more prominent since the guide already requires knowledge about git(hub) without linking any ressources.

Reply:
So you suggest to incorporate branch structure inline into "Getting started"?

Reply:
No, I don't understand how my comment could be interpreted that way? I was just talking about a link to a guide about how to set up a fork.

Reply:
@eps1lon I maybe entered this comment https://github.com/mui-org/material-ui/pull/17562#discussion_r328015796a in the wrong reply box.

It was my reply on #17562 (review).